apb.yml

```diff
@@ -53,37 +53,37 @@ plans:
         required: True
         title: Sync App Port
       - name: SYNC_APP_GRAPHQL_ENDPOINT
-        description: GraphQL Endpoint of your Sync Application (using 'graphql' if not specified)
+        description: (default value: 'graphql') GraphQL Endpoint of your Sync Application 
         type: string
         required: False
         title: Sync App GraphQL Endpoint
       - name: DB_HOSTNAME
-        description: Hostname of the SQL Server
+        description: Hostname of your Database Server
```

Comment:
maybe just say `Hostname of the Database Server`? If we say `your Database` here, then let's keep using the same in the descriptions for the configurations below.